{u'description': [u'VMACS Personal Gentoo Overlay'],
 u'feeds': [u'https://cgit.gentoo.org/user/vmacs.git/atom/'],
 u'homepage': u'https://cgit.gentoo.org/user/vmacs.git/',
 u'irc': None,
 u'name': u'vmacs',
 u'official': False,
 u'owner_email': u'vance.m.allen@gmail.com',
 u'owner_name': u'Vance M. Allen',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://anongit.gentoo.org/user/vmacs.git', u'Git', u''),
              (u'https://anongit.gentoo.org/git/user/vmacs.git', u'Git', u''),
              (u'git+ssh://git@git.gentoo.org/user/vmacs.git', u'Git', u'')],
 u'status': u'unofficial',
 u'supported': True}
$ pmaint sync vmacs
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing vmacs
*** synced vmacs
 * Sync succeeded
$ pmaint regen -t 2 vmacs
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Error running an operation:
 Unhandled exception in regen_cache
Exception pkgcore.ebuild.processor.TimeoutError: TimeoutError() in <functools.partial object at 0x7f4086082578> ignored
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in sys.exitfunc:
ERROR pkgcore: 
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
pkgcore.ebuild.processor.TimeoutError
 * Cache regen failed with 246
